website/common/script/ops/scoreTask.js

```diff
@@ -197,7 +197,7 @@ module.exports = function scoreTask (options = {}, req = {}) {
 
   // This is for setting one-time temporary flags, such as streakBonus or itemDropped. Useful for notifying
   // the API consumer, then cleared afterwards
-  user._tmp = {};
+  if (!user._tmp) user._tmp = {};
```

Comment:
this way it'll never be cleared?

Reply:
Is this field even saved to the database? From my testing it looked like it wasn't saved.

Reply:
it's not saved but what happens if when bulk scoring 2 tasks both return a drop? only the first one will be added to user._tmp, the other will be ignored

Reply:
Yes. That is the part I meant I'm not yet happy with. I'm still thinking about how to solve this while still maintaining backwards compatability. I'm thinking about adding a separate field that is a list of _tmp fields, that the API could return when bulk scored?

Reply:
Yeah so that's why I suggested making a simpler version even if it results in more save requests, right now the API for scoring is geared towards one single task and I feel like it would take some time to correctly plan a new version that works well with multiple ones but I haven't looked in depth